.github/workflows/cicd.yml

```diff
@@ -77,7 +77,6 @@ jobs:
         run: |
           export VERSION=$(cat VERSION)
           export BRANCH=$(cat BRANCH)
-          git pull --rebase
           make generate-operator-yaml
           git config --global user.name ${{ secrets.ACTIONS_USER }}
```

Comment:
Why did we make the user and email secrets?

Reply:
Not sure, that was @raghu-sarangapani, a lot of this code stayed the same